dor/adapters/bag_adapter.py

```diff
@@ -3,30 +3,53 @@
 
 import bagit 
 
+
 class DorInfoMissingError(Exception):
     pass
 
-class BagReader:
+
+class ValidationError(Exception):
+
+    def __init__(self, message: str):
+        super().__init__(message)
+
+        self.message = message
+
+
+class FakeBagAdapter():
+
+    def __init__(self, identifier: str) -> None:
+        self.identifier = identifier
+
+    def validate(self) -> None:
+        return
+    
+    @property
+    def dor_info(self) -> dict:
+        return {
+            'Root-Identifier': "00000000-0000-0000-0000-000000000001"
+        }
+
+
+class BagAdapter:
     
     dor_info_file_name = "dor-info.txt"
 
     def __init__(self, path: Path) -> None:
         self.bag = bagit.Bag(str(path))
 
-    def is_valid(self):
-        is_valid = True
-
+    def validate(self) -> None:
         try:
             self.bag.validate()
         except bagit.BagValidationError as e:
-            is_valid = False
-            print(e)
-            
+            raise ValidationError(f"Validation failed with the following message: \"{str(e)}\"")
```

Comment:
`str()` might not be needed here, but I do want to call `bagit-python`'s `__str__` method which captures all the error details in a single string. So being extra explicit I guess?